integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/allqueries/AllDataTypesTestCase.scala

```diff
@@ -73,6 +75,39 @@ class AllDataTypesTestCase extends QueryTest with BeforeAndAfterAll {
     }
   }
 
+  test("insert data with from_unixtime(0) and query") {
+    sql("drop table if exists time_carbon1")
+    sql("drop table if exists time_parquet")
+    sql("create table if not exists time_carbon1(time1 timestamp) stored as carbondata")
+    sql("create table if not exists time_parquet(time1 timestamp) stored as parquet")
+    sql("insert into time_carbon1 select from_unixtime(0)")
+    sql("insert into time_parquet select from_unixtime(0)")
+    checkAnswer(sql("select * from time_carbon1"),
+      sql("select * from time_parquet"))
+    checkAnswer(sql("select count(*) from time_carbon1"), Seq(Row(1)))
+    checkAnswer(sql("select * from time_carbon1"),
```

Comment:
it is sufficient to check answer with parquet table itslef. you can remove this checkanswer

Reply:
ok